.github/workflows/update-release.yml

```diff
@@ -0,0 +1,20 @@
+name: Update Release Branch
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  update-release-branch:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Get latest tag
+        run: |
+          LATEST_TAG=$(git describe --tags `git rev-list --tags --max-count=1`)
+          echo "LATEST_TAG=${LATEST_TAG}" >> $GITHUB_ENV
+      - name: Update release branch
+        run: |
+          git checkout release
+          git reset --hard $LATEST_TAG
+          git push origin release --force
```

Comment:
> And I don't think we should use --force, the workflow should fail if there's something wrong with the branch.

Agreed.

Can't we simply push directly?

```bash
$ git push origin ${{github.ref_name}}:release
```

From `git-push(1)`:

```
<refspec>...
           Specify what destination ref to update with what source object. The
           format of a <refspec> parameter is an optional plus +, followed by
           the source object <src>, followed by a colon :, followed by the
           destination ref <dst>.

           The <src> is often the name of the branch you would want to push,
           but it can be any arbitrary "SHA-1 expression", such as master~4 or
           HEAD (see gitrevisions(7)).
```

With my suggestion, `${{github.ref_name}}` must always be non-empty. Otherwise, this will simply remove the remote branch. Since we would trigger that only for tag pushes, it does not seem an issue.

Reply:
@henriquesimoes this works just fine, thanks for the suggestion! :)